b2sdk/account_info/abstract.py

```diff
@@ -329,3 +331,33 @@ def clear_large_file_upload_urls(self, file_id):
         :param str file_id: a file ID
         """
         pass
+
+    # TODO: Remove when s3ApiUrl is returned by the server. See #200 for details.
+    @classmethod
+    def _construct_s3_api_url(cls, api_url: str) -> Optional[str]:
```

Comment:
This may change as I'm still waiting for information what are the correct endpoints for non-production realms